### PR TITLE
docs: ng-conf 2020 campaign date change

### PR DESCRIPTION
**Pull request by aaronfrost**
Update the range of dates where ng-conf will appear on angular.io. The current dates won't start until March 1st. But in speaking with @StephenFluin we want it to start sooner. ASAP. 

We also want this PR merged back into the v8 branch so that it lands on angular.io immediately. 

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

### CODE REVIEW

File: aio/content/marketing/announcements.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [
   {
-    "startDate": "2020-03-01",
-    "endDate": "2020-03-31",
+    "startDate": "2020-01-01",
+    "endDate": "2020-04-04",
```
Comment (gkalpak) [inline aio/content/marketing/announcements.json]: We have a different notification for watching the live stream between April 1-3 (see below), so we probably shouldn't be showing that.

```suggestion
    "endDate": "2020-03-31",
```

Comment (IgorMinar) [reply aio/content/marketing/announcements.json]: I agree

Comment (IgorMinar) [reply aio/content/marketing/announcements.json]: @aaronfrost can you please update the PR with a fixup commit? thanks

**Review by IgorMinar — APPROVED** [review]

**Comment by aaronfrost** [comment]
Changes from @gkalpak have been accepted. Let me know if anything else is needed to merge this.

**Review by StephenFluin — APPROVED** [review]

**Comment by gkalpak** [comment]
@googlebot I consent.

**Comment by gkalpak** [comment]
The `lint` CI job was failing, because the second commit did not follow our [commit message guidelines][1]. I fixed it for you.

The `test_docs_examples` failure is a flake. Created #35038 to track it (as I've seen it a couple of time lately).

Also, rebased on master. This should be good to go now.

[1]: https://github.com/angular/angular/blob/d43187f7/CONTRIBUTING.md#commit

**Comment by gkalpak** [comment]
@caretaker, this needs to be merged to master, patch (9.0.x) and 8.2.x.

**Comment by AndrewKushnir** [comment]
@gkalpak there was a conflict merging these changes to 8.2.x branch, since we have 2019 ng-conf there: https://github.com/angular/angular/blob/8.2.x/aio/content/marketing/announcements.json#L5.

Could you plz create a PR that targets 8.2.x branch with the necessary changes? Thank you.

**Pull request closed without merging**